src/main/java/seedu/address/logic/commands/tag/DeleteCommand.java

```diff
@@ -49,12 +50,12 @@ public CommandResult execute(Model model) throws CommandException {
             model.deleteSaleTag(tagToDelete);
             model.updateFilteredPersonList(PREDICATE_SHOW_ALL_PERSONS);
             return new CommandResult(String.format(MESSAGE_DELETE_TAG_SUCCESS, tagToDelete));
+        } else {
+            Tag tagToDelete = contactTagList.get(targetIndex.getZeroBased());
+            model.deleteContactTag(tagToDelete);
+            model.updateFilteredPersonList(PREDICATE_SHOW_ALL_PERSONS);
+            return new CommandResult(String.format(MESSAGE_DELETE_TAG_SUCCESS, tagToDelete));
```

Comment:
p.s. i think these 2 lines are common across the conditionals. maybe we can extract it out of the conditional? same for `EditCommand`